src/lib.rs

```diff
@@ -656,11 +657,12 @@ impl<T: Clone> Sender<T> {
     /// assert_eq!(s.broadcast(2).await, Err(SendError(2)));
     /// # });
     /// ```
-    pub fn broadcast(&self, msg: T) -> Send<'_, T> {
+    pub fn broadcast(&self, msg: T) -> Send<'static, T> {
         Send {
-            sender: self,
+            sender: self.clone(),
```

Comment:
Is this clone expensive? My main concern would be if this could cause a performance degradation.

Reply:
@notgull it's an `Arc` so it really shouldn't be but there is always a tiny bit of performance penalty to pay for any atomic operation so I'm actually thinking this PR isn't a great idea, given [that current API already provides a way forward](https://github.com/smol-rs/async-broadcast/issues/39#issuecomment-1454900141).